Services/Interfaces/web_interface/__init__.py

```diff
@@ -49,6 +51,7 @@ def register_notifier(notification_key, notifier):
 
 # Override system configuration content types
```

Comment:
Can you move this comment next to `flask_util.init_content_types()` please? Because that's the propre place for it 😉

Reply:
like this ?